web_app/rest_client/tasks.py

```diff
@@ -0,0 +1,50 @@
+import requests

```

Comment:
I think, better to use Class for rest_client
and set parameters like user_id during init
Please, cahnge you implementation or create task for it

Reply:
https://github.com/stasya72008/highway-to-hell/issues/24